CODEOWNERS

```diff
@@ -1,16 +1,17 @@
 * @reset @fnichol @christophermaier
-components/backline @fnichol
+components/backline @fnichol @baumanj
 components/builder* @chefsalim @raskchanky
 components/butterfly* @christophermaier @reset
-components/common @fnichol
+components/common @fnichol @baumanj
 components/eventsrv* @reset @raskchanky
-components/hab* @reset @fnichol @christophermaier @chefsalim @elliott-davis @raskchanky
-components/launcher* @reset
-components/net @reset
+<<<<<<< HEAD
```

Comment:
Whoops

Reply:
Facepalm. Thanks for catching this; fixed now.